src/server/shared/DataStores/DBCStructure.h

```diff
@@ -851,386 +856,399 @@ struct HolidayNamesEntry
 
 struct HolidaysEntry
 {
-    uint32 Id;                                              // 0        m_ID
-    uint32 Duration[MAX_HOLIDAY_DURATIONS];                 // 1-10     m_duration
-    uint32 Date[MAX_HOLIDAY_DATES];                         // 11-36    m_date (dates in unix time starting at January, 1, 2000)
-    uint32 Region;                                          // 37       m_region (wow region)
-    uint32 Looping;                                         // 38       m_looping
-    uint32 CalendarFlags[MAX_HOLIDAY_FLAGS];                // 39-48    m_calendarFlags
-    //uint32 holidayNameId;                                 // 49       m_holidayNameID (HolidayNames.dbc)
-    //uint32 holidayDescriptionId;                          // 50       m_holidayDescriptionID (HolidayDescriptions.dbc)
-    char* TextureFilename;                                  // 51       m_textureFilename
-    uint32 Priority;                                        // 52       m_priority
-    int32 CalendarFilterType;                               // 53       m_calendarFilterType (-1 = Fishing Contest, 0 = Unk, 1 = Darkmoon Festival, 2 = Yearly holiday)
-    //uint32 flags;                                         // 54       m_flags (0 = Darkmoon Faire, Fishing Contest and Wotlk Launch, rest is 1)
+    uint32 ID;                                              // 0
+    uint32 Duration[MAX_HOLIDAY_DURATIONS];                 // 1-10
+    uint32 Date[MAX_HOLIDAY_DATES];                         // 11-36 dates in unix time starting at January, 1, 2000
+    uint32 Region;                                          // 37 WoW region
+    uint32 Looping;                                         // 38
+    uint32 CalendarFlags[MAX_HOLIDAY_FLAGS];                // 39-48
+    //uint32 HolidayNameID;                                 // 49 HolidayNames.dbc
+    //uint32 HolidayDescriptionID;                          // 50 HolidayDescriptions.dbc
+    char* TextureFilename;                                  // 51
+    uint32 Priority;                                        // 52
+    int32 CalendarFilterType;                               // 53
+    //uint32 Flags;                                         // 54 0 = Darkmoon Fair, Fishing Contest and Wotlk Launch, rest is 1
```

Comment:
heh, is this some ide-integrated spellchecker? it failed, the name is "Darkmoon Faire" (HolidayNames.dbc)

Reply:
Hrm, no, that was the "me"-integrated spell checker ;)